doc/modelling/reaction/michaelis_menten_kinetics.rst

```diff
@@ -16,29 +15,73 @@ where :math:`S` is the stoichiometric matrix and the fluxes are given by
 .. math::
 
     \begin{aligned}
-        \nu_j = \frac{\mu_{\mathrm{max},j} \, c_S}{k_{\mathrm{MM},j} + c_S},
+        \nu_{i,j} = \prod_{i = 1}^{N_{sub}} \frac{\mu_{\mathrm{max},j} \, c_i}{k_{\mathrm{MM},j} + c_i},
     \end{aligned}
 
 where
 
 - :math:`j` is the reaction index,
-- :math:`c_S` is the substrate component,
+- :math:`N_{sub}` is the number of substrates in the reaction,
+- :math:`c_i` is the substrate component,
 - :math:`\mu_{\mathrm{max},j}`, is the limiting rate approached by the system at saturation,
-- :math:`k_{\mathrm{MM},j}` is the Michaelis constant, which is defined as the concentration of substrate at which the reaction rate is half ov :math:`\mu_{\mathrm{max},j}`.
+- :math:`k_{\mathrm{MM},j}` is the Michaelis constant, which is defined as the concentration of substrate at which the reaction rate is half of :math:`\mu_{\mathrm{max},j}`.
+
+
+In addition, a substrate :math:`i` might be inhibited by other components.
+Three types of inhibition are implemented.
+In this case the rate :math:`\nu_{i,j}` is modified as follows:
```

Comment:
```suggestion
In this case the flux :math:`\nu_{i,j}` is modified as follows:
```

Reply:
Tagging @lieres

Reply:
For the [MAL](https://cadet.github.io/master/modelling/reaction/mass_action_law.html), we use $\varphi$ for the flux, so I suggest to also use this here.